[CI] Failure in SearchableSnapshotsCanMatchOnCoordinatorIntegTests

**Build scan**:

https://gradle-enterprise.elastic.co/s/vh5o7dmperuqw

**Repro line**:

./gradlew ':x-pack:plugin:searchable-snapshots:internalClusterTest' --tests "org.elasticsearch.xpack.searchablesnapshots.SearchableSnapshotsCanMatchOnCoordinatorIntegTests.testQueryPhaseIsExecutedInAnAvailableNodeWhenAllShardsCanBeSkipped" \
  -Dtests.seed=6C796CC2FC93A9F8 \
  -Dtests.security.manager=true \
  -Dtests.locale=nl-NL \
  -Dtests.timezone=Turkey \
  -Druntime.java=11


**Reproduces locally?**:

yes

**Applicable branches**:

master / 7.x / 7.11

**Failure history**:

https://build-stats.elastic.co/goto/be89a3405e2bb1e66cdaaf7bdcdccdf0

It seems it started failing around Dec 15

**Failure excerpt**:

```
  2> java.lang.AssertionError: 
    Expected: not sameInstance(<EMPTY>)
         but: was <EMPTY>
        at __randomizedtesting.SeedInfo.seed([6C796CC2FC93A9F8:3556212E58B7161]:0)
        at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:18)
        at org.junit.Assert.assertThat(Assert.java:956)
        at org.junit.Assert.assertThat(Assert.java:923)
        at org.elasticsearch.xpack.searchablesnapshots.SearchableSnapshotsCanMatchOnCoordinatorIntegTests.testQueryPhaseIsExecutedInAnAvailableNodeWhenAllShardsCanBeSkipped(SearchableSnapshotsCanMatchOnCoordinatorIntegTests.java:310)

```

--- a/x-pack/plugin/searchable-snapshots/src/internalClusterTest/java/org/elasticsearch/xpack/searchablesnapshots/SearchableSnapshotsCanMatchOnCoordinatorIntegTests.java
+++ b/x-pack/plugin/searchable-snapshots/src/internalClusterTest/java/org/elasticsearch/xpack/searchablesnapshots/SearchableSnapshotsCanMatchOnCoordinatorIntegTests.java
@@ -98,7 +98,7 @@ public class SearchableSnapshotsCanMatchOnCoordinatorIntegTests extends BaseSear
         // Either add data outside of the range, or documents that don't have timestamp data
         final boolean indexDataWithTimestamp = randomBoolean();
         if (indexDataWithTimestamp) {
-            indexDocumentsWithTimestampWithinDate(indexOutsideSearchRange, between(0, 1000), "2020-11-26T%02d:%02d:%02d.%09dZ");
+            indexDocumentsWithTimestampWithinDate(indexOutsideSearchRange, between(1, 1000), "2020-11-26T%02d:%02d:%02d.%09dZ");
         } else {
             indexRandomDocs(indexOutsideSearchRange, between(0, 1000));
         }
@@ -222,7 +222,6 @@ public class SearchableSnapshotsCanMatchOnCoordinatorIntegTests extends BaseSear
         }
     }
 
-    @AwaitsFix(bugUrl = "https://github.com/elastic/elasticsearch/issues/66774")
     public void testQueryPhaseIsExecutedInAnAvailableNodeWhenAllShardsCanBeSkipped() throws Exception {
         internalCluster().startMasterOnlyNode();
         internalCluster().startCoordinatingOnlyNode(Settings.EMPTY);
@@ -240,7 +239,7 @@ public class SearchableSnapshotsCanMatchOnCoordinatorIntegTests extends BaseSear
                 .build()
         );
 
-        indexDocumentsWithTimestampWithinDate(indexOutsideSearchRange, between(0, 1000), "2020-11-26T%02d:%02d:%02d.%09dZ");
+        indexDocumentsWithTimestampWithinDate(indexOutsideSearchRange, between(1, 1000), "2020-11-26T%02d:%02d:%02d.%09dZ");
 
         final String repositoryName = randomAlphaOfLength(10).toLowerCase(Locale.ROOT);
         createRepository(repositoryName, "mock");

